server/db/mysql/adapter.go

```diff
@@ -297,7 +297,7 @@ func (a *adapter) CreateDb(reset bool) error {
 			stateat   DATETIME(3),
 			touchedat DATETIME(3),
 			name      CHAR(25) NOT NULL,
-			usebt     INT DEFAULT 0,
+			usebt     TINYINT DEFAULT 0,
```

Comment:
Consider leaving a comment to clarify what this bit signifies.

Reply:
Added comments to rethindb/schema.md. MySQL does not have comments right now. Maybe it should reference rethink instead of duplicating comments there.